### ISSUE TITLE
The MultiSelect doesn't have an aria-required definition

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-react#1473]
## I'm submitting a...
* Bug report 

## Current behavior
After the latest accessibility improvements available with R1 2023 of the KendoReact library, the **aria-required** attribute is missing in most of the DropDowns. 

## Expected behavior
The **aria-required** attribute should be rendered in the DOM once the MultiSelect definition contains a **required** attribute like this:
```
  <MultiSelect required data={sports} onChange={onChange} value={value} />
```

## Minimal reproduction of the problem with instructions
1. Open [this StackBlitz example](https://stackblitz.com/edit/react-zhhyus-tknhmh?file=app%2Fmain.jsx)
2. Inspect the MultiSelect component to see that the **aria-required** in not rendered

**Comment by iadnonov** [comment]
The issue has been fixed in our development channel version `v5.11.0-dev.202301261316`. See the following article about obtaining the development builds - https://www.telerik.com/kendo-react-ui/components/installation/development-builds/.

**Comment by nstoychev** [comment]
We published v5.11.0.
